stitching/stitcher.py

```diff
@@ -96,7 +105,7 @@ def stitch(self, img_names):
         self.estimate_scale(cameras)
 
         imgs = self.resize_low_resolution(imgs)
-        imgs, masks, corners, sizes = self.warp_low_resolution(imgs, cameras)
+        imgs, masks, corners, sizes = self.warp_low_resolution(imgs, cameras, feature_masks)
```

Comment:
we don't need the feature masks in low resolution

Reply:
Discussion in https://github.com/lukasalexanderweber/stitching/pull/55#issuecomment-1457960716